core/src/main/java/org/commonjava/indy/core/change/StoreContentListener.java

```diff
@@ -75,18 +75,32 @@
     @Inject
     private DirectContentAccess directContentAccess;
 
+    /**
+     * Handles store disable/enablement.
+     */
     @IndyMetrics( measure = @Measure( timers = @MetricNamed( name =
                     IndyMetricsCoreNames.METHOD_STORECONTENTLISTENER_ONSTOREDISABLE
                                     + IndyMetricsNames.TIMER ), meters = @MetricNamed( name =
                     IndyMetricsCoreNames.METHOD_STORECONTENTLISTENER_ONSTOREDISABLE + IndyMetricsNames.METER ) ) )
-    public void onStoreDisable( @Observes final ArtifactStoreEnablementEvent event )
+    public void onStoreEnablement( @Observes final ArtifactStoreEnablementEvent event )
```

Comment:
If we don't remove these on disable, won't we still see metadata entries from the disabled repository? Why wait for re-enable?